### ISSUE TITLE
Pin code 1236 is not working to get logged in 

### ISSUE THREAD

**Issue by sumitchauhanJan** [vippsas/vipps-developers#136]
As per documentation for testing  Under **Test and go live** in  [https://docs.adyen.com/payment-methods/vipps/android-drop-in#test-and-go-live](url) the pin 1236 is not working.

**Comment by cloveras** [comment]
Please see our documentation: https://github.com/vippsas/vipps-developers/blob/master/vipps-test-environment.md#vipps-test-apps

**Comment by sumitchauhanJan** [comment]
Yeah ,but Pin 1236 is not working, it used to work few days back.

**Comment by cloveras** [comment]
Can you please explain _exactly_ what the problem is?

**Comment by sumitchauhanJan** [comment]
I have an installed Vipps MT app which is used for testing and now to login to app it requires a Pin .
The pin mentioned 1236 in  https://github.com/vippsas/vipps-developers/blob/master/vipps-test-environment.md#vipps-test-apps is not working ,attaching a screen recording of app
https://user-images.githubusercontent.com/97955648/166225141-42815a34-4cd1-47c9-9d1a-472579d15627.mp4

.

**Comment by cloveras** [comment]
Ok, so you did manage to install the app and use "1236" as documented?

If you use one of the test numbers in Adyen's documentation: Could it be that another Adyen customer has used the same test number and changed the PIN? If so: Please contact Adyen. Vipps never changes the PIN.

**Comment by cloveras** [comment]
I'm closing this due to inactivity. Hope it worked out!
